stdlib/src/main/scala/chisel3/std/tilelink/Parameters.scala

```diff
@@ -0,0 +1,91 @@
+package chisel3.std.tilelink
+
+object TLBundleParameter {
+  /**
+    * Convenient method to generate a unified [[TLBundleParameter]] for typical TL-U bus
```

Comment:
Need to give a better explanation to "typical", for example address and data width are same.